### PR TITLE
Adding Itead Range Extender

### LINKED ISSUE #2106: Using ZBDongle-E in router mode, getting info in the logs to report the issue here.
I am using ZBDongle-E (plugged in my HomeAssistant PC) for a coordinator in ZHE. And using another ZBDongle-E, flashed in router mode, as a router to extend range. Because all of my end devices are battery powered, I used a spare ZBDongle-E to make a router.

I am getting this in HA logs:
```
Logger: homeassistant.components.zha.core.channels.base
Source: components/zha/core/channels/base.py:486
Integration: Zigbee Home Automation (documentation, issues)
First occurred: 15 January 2023 at 18:12:34 (2 occurrences)
Last logged: 15 January 2023 at 18:12:34

[0x73B0:2:0x0300]: [Min mireds is 0, setting to 153] Please open an issue on the quirks repo to have this device corrected
```

And for this reason I am posting here.

<details>

<summary>Device signature</summary>

```yaml
{
  "node_descriptor": "NodeDescriptor(logical_type=<LogicalType.Router: 1>, complex_descriptor_available=0, user_descriptor_available=0, reserved=0, aps_flags=0, frequency_band=<FrequencyBand.Freq2400MHz: 8>, mac_capability_flags=<MACCapabilityFlags.AllocateAddress|RxOnWhenIdle|MainsPowered|FullFunctionDevice: 142>, manufacturer_code=4742, maximum_buffer_size=82, maximum_incoming_transfer_size=82, server_mask=11264, maximum_outgoing_transfer_size=82, descriptor_capability_field=<DescriptorCapability.NONE: 0>, *allocate_address=True, *is_alternate_pan_coordinator=False, *is_coordinator=False, *is_end_device=False, *is_full_function_device=True, *is_mains_powered=True, *is_receiver_on_when_idle=True, *is_router=True, *is_security_capable=False)",
  "endpoints": {
    "1": {
      "profile_id": 260,
      "device_type": "0x0101",
      "in_clusters": [
        "0x0000",
        "0x0003",
        "0x0004",
        "0x0005",
        "0x0006",
        "0x0008"
      ],
      "out_clusters": []
    },
    "2": {
      "profile_id": 260,
      "device_type": "0x010d",
      "in_clusters": [
        "0x0000",
        "0x0003",
        "0x0004",
        "0x0005",
        "0x0006",
        "0x0008",
        "0x0300",
        "0x1000"
      ],
      "out_clusters": []
    },
    "242": {
      "profile_id": 41440,
      "device_type": "0x0061",
      "in_clusters": [],
      "out_clusters": [
        "0x0021"
      ]
    }
  },
  "manufacturer": "SONOFF",
  "model": "DONGLE-E_R",
  "class": "zigpy.device.Device"
}
```

</details>

<details>

<summary>Diagnostic information</summary>

```yaml
{
  "home_assistant": {
    "installation_type": "Home Assistant Container",
    "version": "2023.1.2",
    "dev": false,
    "hassio": false,
    "virtualenv": false,
    "python_version": "3.10.7",
    "docker": true,
    "arch": "x86_64",
    "timezone": "Europe/Prague",
    "os_name": "Linux",
    "os_version": "5.15.83-1-pve",
    "run_as_root": true
  },
  "custom_components": {
    "skodaconnect": {
      "version": "v1.2.3",
      "requirements": [
        "skodaconnect>=1.3.4",
        "homeassistant>=2022.12.0"
      ]
    },
    "frigate": {
      "version": "3.0.0",
      "requirements": []
    },
    "hacs": {
      "version": "1.29.1",
      "requirements": [
        "aiogithubapi>=22.10.1"
      ]
    },
    "uponor": {
      "version": "0.8.1",
      "requirements": [
        "UponorJnap==0.1"
      ]
    }
  },
  "integration_manifest": {
    "domain": "zha",
    "name": "Zigbee Home Automation",
    "config_flow": true,
    "documentation": "https://www.home-assistant.io/integrations/zha",
    "requirements": [
      "bellows==0.34.6",
      "pyserial==3.5",
      "pyserial-asyncio==0.6",
      "zha-quirks==0.0.90",
      "zigpy-deconz==0.19.2",
      "zigpy==0.53.0",
      "zigpy-xbee==0.16.2",
      "zigpy-zigate==0.10.3",
      "zigpy-znp==0.9.2"
    ],
    "usb": [
      {
        "vid": "10C4",
        "pid": "EA60",
        "description": "*2652*",
        "known_devices": [
          "slae.sh cc2652rb stick"
        ]
      },
      {
        "vid": "1A86",
        "pid": "55D4",
        "description": "*sonoff*plus*",
        "known_devices": [
          "sonoff zigbee dongle plus v2"
        ]
      },
      {
        "vid": "10C4",
        "pid": "EA60",
        "description": "*sonoff*plus*",
        "known_devices": [
          "sonoff zigbee dongle plus"
        ]
      },
      {
        "vid": "10C4",
        "pid": "EA60",
        "description": "*tubeszb*",
        "known_devices": [
          "TubesZB Coordinator"
        ]
      },
      {
        "vid": "1A86",
        "pid": "7523",
        "description": "*tubeszb*",
        "known_devices": [
          "TubesZB Coordinator"
        ]
      },
      {
        "vid": "1A86",
        "pid": "7523",
        "description": "*zigstar*",
        "known_devices": [
          "ZigStar Coordinators"
        ]
      },
      {
        "vid": "1CF1",
        "pid": "0030",
        "description": "*conbee*",
        "known_devices": [
          "Conbee II"
        ]
      },
      {
        "vid": "10C4",
        "pid": "8A2A",
        "description": "*zigbee*",
        "known_devices": [
          "Nortek HUSBZB-1"
        ]
      },
      {
        "vid": "0403",
        "pid": "6015",
        "description": "*zigate*",
        "known_devices": [
          "ZiGate+"
        ]
      },
      {
        "vid": "10C4",
        "pid": "EA60",
        "description": "*zigate*",
        "known_devices": [
          "ZiGate"
        ]
      },
      {
        "vid": "10C4",
        "pid": "8B34",
        "description": "*bv 2010/10*",
        "known_devices": [
          "Bitron Video AV2010/10"
        ]
      }
    ],
    "codeowners": [
      "@dmulcahey",
      "@adminiuga",
      "@puddly"
    ],
    "zeroconf": [
      {
        "type": "_esphomelib._tcp.local.",
        "name": "tube*"
      },
      {
        "type": "_zigate-zigbee-gateway._tcp.local.",
        "name": "*zigate*"
      },
      {
        "type": "_zigstar_gw._tcp.local.",
        "name": "*zigstar*"
      },
      {
        "type": "_slzb-06._tcp.local.",
        "name": "slzb-06*"
      }
    ],
    "dependencies": [
      "file_upload"
    ],
    "after_dependencies": [
      "onboarding",
      "usb",
      "zeroconf"
    ],
    "iot_class": "local_polling",
    "loggers": [
      "aiosqlite",
      "bellows",
      "crccheck",
      "pure_pcapy3",
      "zhaquirks",
      "zigpy",
      "zigpy_deconz",
      "zigpy_xbee",
      "zigpy_zigate",
      "zigpy_znp"
    ],
    "is_built_in": true
  },
  "data": {
    "ieee": "**REDACTED**",
    "nwk": 29616,
    "manufacturer": "SONOFF",
    "model": "DONGLE-E_R",
    "name": "SONOFF DONGLE-E_R",
    "quirk_applied": false,
    "quirk_class": "zigpy.device.Device",
    "manufacturer_code": 4742,
    "power_source": "Mains",
    "lqi": 152,
    "rssi": -62,
    "last_seen": "2023-01-16T15:50:10",
    "available": true,
    "device_type": "Router",
    "signature": {
      "node_descriptor": "NodeDescriptor(logical_type=<LogicalType.Router: 1>, complex_descriptor_available=0, user_descriptor_available=0, reserved=0, aps_flags=0, frequency_band=<FrequencyBand.Freq2400MHz: 8>, mac_capability_flags=<MACCapabilityFlags.AllocateAddress|RxOnWhenIdle|MainsPowered|FullFunctionDevice: 142>, manufacturer_code=4742, maximum_buffer_size=82, maximum_incoming_transfer_size=82, server_mask=11264, maximum_outgoing_transfer_size=82, descriptor_capability_field=<DescriptorCapability.NONE: 0>, *allocate_address=True, *is_alternate_pan_coordinator=False, *is_coordinator=False, *is_end_device=False, *is_full_function_device=True, *is_mains_powered=True, *is_receiver_on_when_idle=True, *is_router=True, *is_security_capable=False)",
      "endpoints": {
        "1": {
          "profile_id": 260,
          "device_type": "0x0101",
          "in_clusters": [
            "0x0000",
            "0x0003",
            "0x0004",
            "0x0005",
            "0x0006",
            "0x0008"
          ],
          "out_clusters": []
        },
        "2": {
          "profile_id": 260,
          "device_type": "0x010d",
          "in_clusters": [
            "0x0000",
            "0x0003",
            "0x0004",
            "0x0005",
            "0x0006",
            "0x0008",
            "0x0300",
            "0x1000"
          ],
          "out_clusters": []
        },
        "242": {
          "profile_id": 41440,
          "device_type": "0x0061",
          "in_clusters": [],
          "out_clusters": [
            "0x0021"
          ]
        }
      }
    },
    "active_coordinator": false,
    "entities": [
      {
        "entity_id": "button.sonoff_dongle_e_r_identify",
        "name": "SONOFF DONGLE-E_R"
      },
      {
        "entity_id": "light.sonoff_dongle_e_r_light",
        "name": "SONOFF DONGLE-E_R"
      },
      {
        "entity_id": "light.sonoff_dongle_e_r_light_2",
        "name": "SONOFF DONGLE-E_R"
      },
      {
        "entity_id": "number.sonoff_dongle_e_r_start_up_current_level",
        "name": "SONOFF DONGLE-E_R"
      },
      {
        "entity_id": "number.sonoff_dongle_e_r_start_up_current_level_2",
        "name": "SONOFF DONGLE-E_R"
      },
      {
        "entity_id": "number.sonoff_dongle_e_r_start_up_color_temperature",
        "name": "SONOFF DONGLE-E_R"
      },
      {
        "entity_id": "select.sonoff_dongle_e_r_start_up_behavior",
        "name": "SONOFF DONGLE-E_R"
      },
      {
        "entity_id": "select.sonoff_dongle_e_r_start_up_behavior_2",
        "name": "SONOFF DONGLE-E_R"
      },
      {
        "entity_id": "sensor.sonoff_dongle_e_r_rssi",
        "name": "SONOFF DONGLE-E_R"
      },
      {
        "entity_id": "sensor.sonoff_dongle_e_r_lqi",
        "name": "SONOFF DONGLE-E_R"
      }
    ],
    "neighbors": [
      {
        "device_type": "Coordinator",
        "rx_on_when_idle": "On",
        "relationship": "Sibling",
        "extended_pan_id": "**REDACTED**",
        "ieee": "**REDACTED**",
        "nwk": "0x0000",
        "permit_joining": "Unknown",
        "depth": "0",
        "lqi": "116"
      },
      {
        "device_type": "EndDevice",
        "rx_on_when_idle": "Off",
        "relationship": "Child",
        "extended_pan_id": "**REDACTED**",
        "ieee": "**REDACTED**",
        "nwk": "0xB9C8",
        "permit_joining": "NotAccepting",
        "depth": "2",
        "lqi": "169"
      },
      {
        "device_type": "EndDevice",
        "rx_on_when_idle": "Off",
        "relationship": "Child",
        "extended_pan_id": "**REDACTED**",
        "ieee": "**REDACTED**",
        "nwk": "0x5A7D",
        "permit_joining": "NotAccepting",
        "depth": "2",
        "lqi": "152"
      },
      {
        "device_type": "EndDevice",
        "rx_on_when_idle": "Off",
        "relationship": "Child",
        "extended_pan_id": "**REDACTED**",
        "ieee": "**REDACTED**",
        "nwk": "0x9DE2",
        "permit_joining": "NotAccepting",
        "depth": "2",
        "lqi": "191"
      },
      {
        "device_type": "EndDevice",
        "rx_on_when_idle": "Off",
        "relationship": "Child",
        "extended_pan_id": "**REDACTED**",
        "ieee": "**REDACTED**",
        "nwk": "0x955A",
        "permit_joining": "NotAccepting",
        "depth": "2",
        "lqi": "128"
      },
      {
        "device_type": "EndDevice",
        "rx_on_when_idle": "Off",
        "relationship": "Child",
        "extended_pan_id": "**REDACTED**",
        "ieee": "**REDACTED**",
        "nwk": "0x0615",
        "permit_joining": "NotAccepting",
        "depth": "2",
        "lqi": "107"
      },
      {
        "device_type": "EndDevice",
        "rx_on_when_idle": "Off",
        "relationship": "Child",
        "extended_pan_id": "**REDACTED**",
        "ieee": "**REDACTED**",
        "nwk": "0x294A",
        "permit_joining": "NotAccepting",
        "depth": "2",
        "lqi": "139"
      }
    ],
    "routes": [
      {
        "dest_nwk": "0x0000",
        "route_status": "Active",
        "memory_constrained": true,
        "many_to_one": true,
        "route_record_required": false,
        "next_hop": "0x0000"
      }
    ],
    "endpoint_names": [
      {
        "name": "DIMMABLE_LIGHT"
      },
      {
        "name": "EXTENDED_COLOR_LIGHT"
      },
      {
        "name": "unknown 97 device_type of 0xa1e0 profile id"
      }
    ],
    "user_given_name": null,
    "device_reg_id": "d58ed94dfc02355b4aeabe15781403ec",
    "area_id": "hodnik",
    "cluster_details": {
      "1": {
        "device_type": {
          "name": "DIMMABLE_LIGHT",
          "id": 257
        },
        "profile_id": 260,
        "in_clusters": {
          "0x0000": {
            "endpoint_attribute": "basic",
            "attributes": {
              "0x0004": {
                "attribute_name": "manufacturer",
                "value": "SONOFF"
              },
              "0x0005": {
                "attribute_name": "model",
                "value": "DONGLE-E_R"
              }
            },
            "unsupported_attributes": {}
          },
          "0x0003": {
            "endpoint_attribute": "identify",
            "attributes": {},
            "unsupported_attributes": {}
          },
          "0x0004": {
            "endpoint_attribute": "groups",
            "attributes": {},
            "unsupported_attributes": {}
          },
          "0x0005": {
            "endpoint_attribute": "scenes",
            "attributes": {},
            "unsupported_attributes": {}
          },
          "0x0006": {
            "endpoint_attribute": "on_off",
            "attributes": {
              "0x0000": {
                "attribute_name": "on_off",
                "value": 0
              },
              "0x4003": {
                "attribute_name": "start_up_on_off",
                "value": 255
              }
            },
            "unsupported_attributes": {}
          },
          "0x0008": {
            "endpoint_attribute": "level",
            "attributes": {
              "0x0000": {
                "attribute_name": "current_level",
                "value": 51
              },
              "0x4000": {
                "attribute_name": "start_up_current_level",
                "value": 51
              }
            },
            "unsupported_attributes": {
              "0x0010": {
                "attribute_name": "on_off_transition_time"
              },
              "0x0011": {
                "attribute_name": "on_level"
              },
              "0x0012": {
                "attribute_name": "on_transition_time"
              },
              "0x0013": {
                "attribute_name": "off_transition_time"
              },
              "0x0014": {
                "attribute_name": "default_move_rate"
              }
            }
          }
        },
        "out_clusters": {}
      },
      "2": {
        "device_type": {
          "name": "EXTENDED_COLOR_LIGHT",
          "id": 269
        },
        "profile_id": 260,
        "in_clusters": {
          "0x0000": {
            "endpoint_attribute": "basic",
            "attributes": {},
            "unsupported_attributes": {}
          },
          "0x0003": {
            "endpoint_attribute": "identify",
            "attributes": {},
            "unsupported_attributes": {}
          },
          "0x0004": {
            "endpoint_attribute": "groups",
            "attributes": {},
            "unsupported_attributes": {}
          },
          "0x0005": {
            "endpoint_attribute": "scenes",
            "attributes": {},
            "unsupported_attributes": {}
          },
          "0x0006": {
            "endpoint_attribute": "on_off",
            "attributes": {
              "0x0000": {
                "attribute_name": "on_off",
                "value": 0
              },
              "0x4003": {
                "attribute_name": "start_up_on_off",
                "value": 255
              }
            },
            "unsupported_attributes": {}
          },
          "0x0008": {
            "endpoint_attribute": "level",
            "attributes": {
              "0x0000": {
                "attribute_name": "current_level",
                "value": 51
              },
              "0x4000": {
                "attribute_name": "start_up_current_level",
                "value": 51
              }
            },
            "unsupported_attributes": {
              "0x0010": {
                "attribute_name": "on_off_transition_time"
              },
              "0x0011": {
                "attribute_name": "on_level"
              },
              "0x0012": {
                "attribute_name": "on_transition_time"
              },
              "0x0013": {
                "attribute_name": "off_transition_time"
              },
              "0x0014": {
                "attribute_name": "default_move_rate"
              }
            }
          },
          "0x0300": {
            "endpoint_attribute": "light_color",
            "attributes": {
              "0x0000": {
                "attribute_name": "current_hue",
                "value": 0
              },
              "0x0001": {
                "attribute_name": "current_saturation",
                "value": 0
              },
              "0x0003": {
                "attribute_name": "current_x",
                "value": 24939
              },
              "0x0004": {
                "attribute_name": "current_y",
                "value": 24701
              },
              "0x0007": {
                "attribute_name": "color_temperature",
                "value": 17476
              },
              "0x0008": {
                "attribute_name": "color_mode",
                "value": 2
              },
              "0x4000": {
                "attribute_name": "enhanced_current_hue",
                "value": 0
              },
              "0x4002": {
                "attribute_name": "color_loop_active",
                "value": 0
              },
              "0x400a": {
                "attribute_name": "color_capabilities",
                "value": 0
              },
              "0x400b": {
                "attribute_name": "color_temp_physical_min",
                "value": 0
              },
              "0x400c": {
                "attribute_name": "color_temp_physical_max",
                "value": 65279
              },
              "0x4010": {
                "attribute_name": "start_up_color_temperature",
                "value": 17476
              }
            },
            "unsupported_attributes": {}
          },
          "0x1000": {
            "endpoint_attribute": "lightlink",
            "attributes": {},
            "unsupported_attributes": {}
          }
        },
        "out_clusters": {}
      },
      "242": {
        "device_type": {
          "name": "unknown",
          "id": 97
        },
        "profile_id": 41440,
        "in_clusters": {},
        "out_clusters": {
          "0x0021": {
            "endpoint_attribute": "green_power",
            "attributes": {},
            "unsupported_attributes": {}
          }
        }
      }
    }
  }
}
```

</details>

**Additional context**
This stick seems to work ok in router mode, because I am seeing that end devices are connected through it. I do see that this ZBDongle-E router is represented in HA as a being some kind of a dual light switch with colors to choose from. There are no switches and lights, of course, it's just a stick, flashed with router firmware, plugged into a USB power supply that's plugged in to a wall socket.

Reason I am posting here is because the logs are instructing me to do so. I guess there's a reason why those messages are logged, so... providing info as requested !

### PR DESCRIPTION
**Pull request by MattWestb**
Deleting not working endpoint and cluster for getting it clean.
Fix #2106

### CODE REVIEW

File: zhaquirks/sonoff/rangeextender.py [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+                INPUT_CLUSTERS: [
+                    Basic.cluster_id,
+                    Identify.cluster_id,
+                ],
+                OUTPUT_CLUSTERS: [],
+            },
+            242: {
+                INPUT_CLUSTERS: [],
+                OUTPUT_CLUSTERS: [33],
+            },
```
Comment (TheJulianJES) [inline zhaquirks/sonoff/rangeextender.py:96]: Is ``PROFILE_ID`` and ``DEVICE_TYPE`` missing in the replacement for the ZGP endpoint?

File: zhaquirks/sonoff/rangeextender.py [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+    ENDPOINTS,
+    INPUT_CLUSTERS,
+    MODELS_INFO,
+    OUTPUT_CLUSTERS,
+    PROFILE_ID,
+)
+
+
+class IteadRangeExtender(CustomDevice):
+    """Itead dongle with range extender firmware devices."""
```
Comment (TheJulianJES) [inline zhaquirks/sonoff/rangeextender.py:27]:
```suggestion
    """Sonoff-E dongle with range extender firmware."""
```

Comment (MattWestb) [reply zhaquirks/sonoff/rangeextender.py]: All Sonoff EFR32MG21 coordinators is having the same pins / pads and can using both firmware.
This is for the Itead firmware and is working on Sonoff, eWeLink and Itead hardware.
Then like quirking the XPS firmware we calling it Sonoff.

Comment (TheJulianJES) [reply zhaquirks/sonoff/rangeextender.py]: Hmm, okay. Suggested Dongle-E because that's how the firmware identifies itself but this is fine too.

**Comment by TheJulianJES** [comment]
Basically looks good now. I do wonder if the two endpoints (with lights) were ever used? E.g., do they control some kind of on-board LED? 

It looks like Z2M exposes some kind of indicator light: https://github.com/Koenkk/zigbee-herdsman-converters/blob/11123422f0fa7085987f8ad6eea8d06b29e62101/devices/sonoff.js#L244

**Comment by Hedda** [comment]
@MattWestb while bikeshedding I am wondering if I may suggest renaming this device handler to include "**ZBDongle-E**" (or "**Dongle-E**") in the name just to separate its name from ITead's Sonoff "**ZBDongle-P**" (also known as "**Dongle-P**") so they are not confused and in order to better future proof a naming scheme? For reference, ITead's product model is called "**ZBDongle-E**" on their website and that is also what is printed on the dongle enclosure, but they currently call their released router firmware "Dongle-E" for some reason:

https://itead.cc/product/zigbee-3-0-usb-dongle/

https://github.com/itead/Sonoff_Zigbee_Dongle_Firmware/tree/master/Dongle-E/Router

PS: I know that  ITead's Sonoff "**ZBDongle-P**" USB dongle today simply uses the generic Z-Stack Router firmware from Koenkk](https://github.com/Koenkk/Z-Stack-firmware) but who knows what will happen in the future, (maybe ITead will start to release their own branded Z-Stack Router firmware, and maybe they change the naming of the EmberZNet Router firmware for the EFR32MG21 adapter to actually reflect the model name ).

**Comment by MattWestb** [comment]
For my is the name the firmware is using is enough `MODELS_INFO: [("SONOFF", "DONGLE-E_R")] `.

**Comment by TheJulianJES** [comment]
Waiting for a reply from someone with a Sonoff-E dongle here: https://github.com/zigpy/zha-device-handlers/issues/2106#issuecomment-1431919458
It seems like the dimmable light entity actually has a use-case and dims the on-board LED (at least Z2M exposes it).

(The "extended color light" seems to still be useless though?)

**Comment by MattWestb** [comment]
One lesser to commuting !!

**Pull request closed without merging**